lib/rumTemplate.js.handlebars

```diff
@@ -1,8 +1,8 @@
 (function() {
 	var beacon = new Image();
-	beacon.src = '//{{{host}}}/v2/recordRumData?X';
+	beacon.src = '//{{{host}}}/v2/recordRumData?X' + 
```

Comment:
I don't think RUM was working before :-O.  I think we may be about to release a broken version in 3.13 unfortunately.